in_toto/util.go

```diff
@@ -132,25 +132,15 @@ func InterfaceKeyStrings(m map[string]interface{}) []string {
 
 /*
 subsetCheck checks if all strings in a slice of strings
-can be found in a superset slice of strings.
+can be found in a Set.
```

Comment:
Please update the docstring, also so that it is clear which one is expected to be the subset and which the superset.